.gitignore

```diff
@@ -110,6 +110,7 @@ $tf/
 _ReSharper*/
 *.[Rr]e[Ss]harper
 *.DotSettings.user
+*.DotSettings
```

Comment:
This should not be excluded. A DotSettings file will be introduced at a later point.